### DISCUSSION TITLE
obsidian 目前最完美的免费发布方案 - 渐进式教程

### DISCUSSION THREAD

**Discussion by giscus** [oldwinter/knowledge-garden#30] [备发布站的评论系统]
# obsidian 目前最完美的免费发布方案 - 渐进式教程

  可以访问此文的双链版本，获得完整阅读体验：obsidian 目前最完美的免费发布方案 - 渐进式教程 - 🌲 oldwinterの数字花园

https://notes.oldwinter.top/obsidian-%E7%9B%AE%E5%89%8D%E6%9C%80%E5%AE%8C%E7%BE%8E%E7%9A%84%E5%85%8D%E8%B4%B9%E5%8F%91%E5%B8%83%E6%96%B9%E6%A1%88-%E6%B8%90%E8%BF%9B%E5%BC%8F%E6%95%99%E7%A8%8B

**Comment by shipengfish** [comment]
thanks
